Validation/HGCalValidation/test/MakePlots.C

```diff
@@ -453,7 +453,7 @@ void hgcalBHValidPlots(std::string fname = "roots/hgcBHValidD83.root",
 
 void ehcalPlots(std::string fname = "ecalHitdd4hep.root",
```

Comment:
It seems the script to update does not cover "dd4hep".
For example, the filename mentioned in Validation/Geometry/test/runP_ECAL_dd4hep_cfg.py has been updated from "DD4HEP" to "DD4hep", i.e. matbdg_ECAL_DD4HEP.root, matbdg_ECAL_DD4HEP.txt

Will changing of filenames affect something else from the DPG side?

Reply:
IMHO, usage of `dd4hep` as a namespace or as an Era is fine. So is usage of `DD4HEP` in macros or preprocessor definitions.

Other inconsistent names include `ECAL`, `ECal` and `Ecal`, etc.